crates/locksmith/src/tracker.rs

```diff
@@ -87,6 +92,20 @@ Backtrace at the moment of guard creation follows:
             annotation=annotation,
             backtrace=self.backtrace
         );
+        #[cfg(not(feature = "debug_info"))]
+        error!(
+            r"
+
+        !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+        !!! IMMORTAL LOCK GUARD FOUND !!!
+        !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+{type:?} guard {puid} lived for > {time} seconds.{annotation}",
+            type=self.lock_type,
+            puid=self.puid,
+            time=IMMORTAL_TIMEOUT.as_secs(),
+            annotation=annotation
+        );
```

Comment:
What is purpose of this new `error!` block here?

Reply:
Conditionally compiled, wanted to provide an error block that doesn't mention the backtrace and just having two blocks of code didn't seem too bad the macro + flag could catch them but could always clean this up